### DISCUSSION TITLE
👋 Hello My bf is loving your tool using [Sep 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3907] [Discord Threads]
> 🙋‍♂️ **Jess (she/her)**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### <@212548363529355264> : thank you so much for your interest .. we would love to get some help on cypress. Will DM in twitter.  
#  
> 🙋‍♂️ **Jess (she/her)**  
#### OK sweet  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020523862783434843)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
